src/main/content/_assets/js/collections.js

```diff
@@ -20,20 +21,37 @@ function getCollectionData(instanceName){
     if(typeof instanceName === "undefined"){
         return;
     }
-
     return fetch(`/api/auth/kabanero/${instanceName}/collections/list`)
         .then(function(response) {
             return response.json();
         })
+        .then(function(data){
+            return updateCollectionView(data);
+        })
+        .then(() =>{
+            return instanceName;
+        })
         .catch(error => console.error("Error getting collections", error));
 }
 
+function getCliVersion(instanceName){
+    if(typeof instanceName === "undefined"){
+        return;
+    }
+    return fetch(`/api/auth/kabanero/${instanceName}/collections/version`)
+        .then(function(response) {
```

Comment:
would probably look more like `.then(setCLIVersion) {` when the first comment about the promise chain is addressed.

I changed the name from `updateCliVersion` -> `setCLIVersion`